server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -56,6 +55,7 @@ import EndOfServiceReportConfirmationView from './endOfServiceReportConfirmation
 import ControllerUtils from '../../utils/controllerUtils'
 import AuthUserDetails from '../../models/hmppsAuth/authUserDetails'
 import ServiceCategory from '../../models/serviceCategory'
+import FinaliseActionPlanActivitiesForm from './finaliseActionPlanActivitiesForm'
```

Comment:
Why has this import moved?

Reply:
I removed it then put it back in and it auto-added here. Have fixed this up.